nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/custom/ReverseSequence.java

```diff
@@ -59,6 +60,14 @@ public ReverseSequence(SameDiff sameDiff, SDVariable i_v, SDVariable seqLengths)
         addArguments();
     }
 
+    public ReverseSequence(INDArray x, INDArray seq_lengths, int seqDim, int batchDim){
+        //TODO: Implement.
+    }
+
+    public ReverseSequence(INDArray x, INDArray seq_lengths){
+        //TODO: Implement.
+    }
```

Comment:
`this(x, seq_lengths, 1, 0)`

Reply:
Fixed. (required a fix to ReverseSequence)